app/graphql/types/record_type.rb

```diff
@@ -126,6 +126,19 @@ class LocationType < Types::BaseObject
       field :kind, String
       field :value, String
     end
+
+    class HighlightType < Types::BaseObject
+      field :kind, String
+      field :value, [String]
```

Comment:
Similarly, I wonder if `:matched_phrases` or something more descriptive than `:value` would be more useful...